src/ui/EastRegion.js

```diff
@@ -390,7 +390,7 @@ EastRegion = function(c) {
                     xtype: 'panel',
                     bodyStyle: 'border-style:none;',
                     id: 'commentContent-' + comment.id,
-                    cls: 'comment greyBackground',
+                    cls: 'comment commentView greyBackground',
```

Comment:
This new class provides just an identifier that will be used by jquery when the user tries to expand all the comment, correct?

Reply:
Yes, though using directly ext.js: `panel.getEl().query(".commentView")`. It was not possible to set all the box as visible because there are hidden edit textarea's with every comment.